style.css

```diff
@@ -0,0 +1,131 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
+
+@media only screen and (max-width: 768px) and (min-width: 375px) {
+  body {
+    width: 100%;
+    margin: auto;
+  }
+
+  .myInfo {
+    width: 100vw;
+    height: 100vh;
+  }
+}
+
+@media only screen (min-width: 768px) {
+  body {
+    width: 100%;
+    margin: auto;
+  }
+
+  .myInfo {
+    width: 100vw;
+    height: 100vh;
+  }
+}
+
+.appbar {
+  position: relative;
+  width: 100%;
+  height: 7rem;
+  left: 0;
+  top: 0;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  left: 8%;
+  right: 8%;
+  top: 5%;
+  bottom: 0;
+  padding-top: 2rem;
+}
+
+.mylogo {
+  width: 30%;
+  height: 2rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.2rem;
+  line-height: 10%;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.hamburger {
+  color: #6070ff;
+  height: 74%;
+  padding-right: 30%;
+}
+
+.myInfo {
+  gap: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Concerning the responsiveness of your project, everything seems perfect, you're doing a great job, except for the background image, as you can see in the image below, as we increase the width, the bkg does not respond, to do so, we have to add another property called `background-size`, go ahead and see how can you make it work.

![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/102612396/dd65d166-0ae4-453f-91f4-45f87e9a7a18)